CH02/ch02_exercises/ch02_ex3.py

```diff
@@ -0,0 +1,6 @@
+def maj_inverser(word):
+    print word.upper() [::-1]
```

Comment:
Use return to return the value  (Also print function should be used with parentheses print('My text') or print(word.upper()[::-1]))